src/components/QuizButton/QuizButton.tsx

```diff
@@ -0,0 +1,32 @@
+import "./QuizButton.css";
+
+interface QuizButtonProps {
+	ingredients: string;
+	onClick: (ingredient: string) => void;
+	ingredientState: "wrong" | "right" | "default";
+}
+
+function QuizButton({
+	ingredients,
+	onClick,
+	ingredientState,
+}: QuizButtonProps) {
+	const buttonStyle =
```

Comment:
https://medium.com/nerd-for-tech/a-case-to-switch-using-switch-statements-in-react-e83e01154f60

Si jamais tu veux tester ça sur ta triple condition avec une valeur par défaut